assets/prometheus-adapter/api-service.yaml

```diff
@@ -1,6 +1,8 @@
 apiVersion: apiregistration.k8s.io/v1beta1
 kind: APIService
 metadata:
+  annotations:
+    'service.alpha.openshift.io/inject-cabundle:': "true"
```

Comment:
there is a `:` in that string too much, also I don't think you need the quotes

Reply:
Yup. I saw that `:` earlier as well... We do need the string for the key.